blocks/library/embed/index.js

```diff
@@ -129,31 +140,38 @@ registerBlockType( 'core/embed', {
 			const { url, caption } = this.props.attributes;
 			const { setAttributes, focus, setFocus } = this.props;
 
+			if ( fetching ) {
+				return (
+					<div className="blocks-embed__loading">
```

Comment:
The docs could probably do for some clarification, but it's intended that the root element of the return value should simply be `top-level-folder` + `folder-name`,  i.e. `blocks-embed`. The treatment here is one reserved for descendants, or other components in the same folder (not `index.js`).

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming

Would be more appropriate to apply loading as a modifier class:

```jsx
<div className="blocks-embed is-loading">
```